src/Directory.Build.props

```diff
@@ -1,7 +1,9 @@
 <Project>
+    <Import Project="Versions.props" />
+
     <PropertyGroup>
         <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
-        <TargetFramework>net7.0</TargetFramework>
+        <TargetFramework>$(TargetFrameworkVersion)</TargetFramework>
```

Comment:
do not use this property. it's recommended to specify the <TargetFrameworks> properties, even if there's only one.
And when there are a small number of projects, don't add yet another property for managing it.

Reply:
note: TargetFrameworkVersion is one of the properties set by the MSBuild targets, or it looks like one that is.
it's used in pre-sdk-style projects, which this is not.